integration/test_dataset.py

```diff
@@ -53,8 +56,87 @@ def test_replace_values(self):
             pass
 
         r = ds.follow("table", "limit=10")["data"]
-        assert r[variable.body["id"]] == [1, 2, 3, 4, 5]
-        ds.delete()
+        try:
+            assert r[variable.body["id"]] == [1, 2, 3, 4, 5]
+        finally:
+            ds.delete()
+
+    def test_append_dataset_any_filter(self):
```

Comment:
```suggestion
    def test_append_dataset_any_filter_multiple_response(self):
```
We'll want another test that does somthing similar by testing `.any()` on a regular categorical.